syntax = "proto3";

package AcFunDanmu.Im.Message;

import "ChatTarget.proto";

message SessionCreateRequest {
	ChatTarget chatTarget = 1;
}
